### ISSUE TITLE
Suggest default # of CPUs (e.g. use default from computer?) and of pools (some simple heuristics?)

### ISSUE THREAD

**Issue by giovannipizzi** [aiidalab/aiidalab-qe#356]
**Is your feature request related to a problem? Please describe.**
By default the app sets 1 CPU and 1 pool (I think, at least for me). I need to be expert to know how many I should use.

**Describe the solution you'd like**
I guess I always want to use the max # of CPUs on the computer (on 1 node) and scale at least to full 1 node by default, and use some pools when memory is not an issue and I have enough k-points.
(BTW, apparently now QE has a auto-parallelisation feature? @mbercx did you try it? We should probably check and use it, at least for the QE app! If you don't have time maybe someone else could try it out @unkcpz or @superstar54 ?)

**Comment by unkcpz** [comment]
> By default the app sets 1 CPU and 1 pool (I think, at least for me). I need to be expert to know how many I should use

This is not true, if you use the remote machine which has the default number of cups set in aiida computer setup, the widget will use the maximum number of CPUs and try to set the proper # of pool. Please see the implementation at 
https://github.com/aiidalab/aiidalab-qe/blob/2e4c7faa50240071e80d855969a49e0643154412/aiidalab_qe/steps.py#L700-L736

> BTW, apparently now QE has a auto-parallelisation feature?

This is definitely worth trying. I'll open an issue for this.
